packages/templates/README.md

```diff
@@ -5,3 +5,9 @@ This package contains the templates used by the dashboard and the editor (as par
 ## Template Creation
 
 See [External Template Creation](../../docs/external-template-creation.md).
+
+## Generating images for templates
+
+_Note: This script requires the development environment to be running with `DISABLE_OPTIMIZED_RENDERING` so the templates are rendered as components not images._
```

Comment:
How so?

Templates are opened in the editor and then previewed on the frontend. `DISABLE_OPTIMIZED_RENDERING` is irrelevant for that, no?

Reply:
😂  ah, i definitely copied this from text-sets documentation and forgot to update it. Caught me!